src/code/rcp_utils.c

```diff
@@ -44,9 +45,11 @@ void RcpUtils_PrintRegisterStatus(void) {
     osSyncPrintf("\n");
 }
 
-void RcpUtils_Reset() {
+void RcpUtils_Reset(void) {
     RcpUtils_PrintRegisterStatus();
+    // flush the RDP command queue and freeze clock counter
     osDpSetStatus(DPC_SET_FREEZE | DPC_SET_FLUSH);
```

Comment:
I believe when freeze and flush are both set the contents of the RDP command FIFO is discarded, but this really warrants a dedicated HW test since as you've noted very little is known about the specifics of what freeze and flush do

Reply:
Although actually, I recall reading before that these two operations act on the RDP "pipeline", so I think what I'd expect to happen is the current RDP command will finish (how quickly this happens depends, for example whether the RDP runs in 1 cycle per pixel or 2 cycle per pixel mode) and then it will freeze the clock counter and not fetch & run any more commands